nvchecker.ini

```diff
@@ -2495,6 +2495,9 @@ aur =
 
 [fcitx5-anthy-git]
 aur =
+
```

Comment:
Add your name and email address here like above. This will be used by lilac in the future to mail error reports.